src/edge/simulator.ts

```diff
@@ -175,6 +175,7 @@ export class Simulator {
 
     public async startEdgeHubSingleModule(outputChannel: vscode.OutputChannel): Promise<void> {
         return await this.callWithInstallationCheck(outputChannel, async () => {
+            await this.checkAndGuideUserToSetupIotedgehubdev(outputChannel);
```

Comment:
from the current flow, it will setup and continue to start?

Reply:
updated